blocks/bio-detail/bio-detail.js

```diff
@@ -0,0 +1,126 @@
+import { div, img } from '../../scripts/dom-helpers.js';
+
+import { fetchLanguagePlaceholders } from '../../scripts/scripts.js';
+
+function createStructure(firstContainer, secondContainer) {
+  const sectionsContainer = div(
+    { class: 'sections-container' },
+    firstContainer,
+    secondContainer,
+  );
+  document.querySelector('.bio-detail.block').append(sectionsContainer);
```

Comment:
Why not pass block as an argument and then append instead of querying the `document`?